autrite/extract_query.py

```diff
@@ -0,0 +1,42 @@
+from loader import Loader
```

Comment:
I think the file is not used, and most contents overlap with benchmark/extract_query.py?
If so, we can just delete this file